R result
ExceptionCallback<Throwable> exceptionCallback
